test/include.test.js

```diff
@@ -478,18 +487,18 @@ describe('include', function() {
                   patient.address(address);
                   patient.save(function() {
                     physician.patients({include: 'address'},
-                      function(err, posts) {
-                        if (err) return done(err);
-
-                        posts.should.be.an.instanceOf(Array).and.have.length(1);
-                        var p = posts[0];
-                        p.name.should.equal('a');
-                        p.addressId.should.equal(1);
-                        p.address().id.should.equal(1);
-                        p.address().name.should.equal('z');
-
-                        done();
-                      });
+                        function(err, patients) {
+                          if (err) return done(err);
+
+                          patients.should.have.length(1);
+                          var p = patients[0];
+                          p.name.should.equal('a');
+                          p.addressId.should.eql(patient.addressId);
+                          p.address().id.should.eql(address.id);
+                          p.address().name.should.equal('z');
+
+                          done();
+                        });
```

Comment:
We can address this as part of homework test refactor. This super nested test needs cleanup anyways.